src/uproot/_dask.py

```diff
@@ -63,6 +64,9 @@ def dask(
             the dask collections. If False, the opening of the files and reading the
             metadata is also delayed till the compute call. In this case, branch-names
             are inferred by opening only the first file.
+        form_mapping (Callable[awkward.forms.form] -> awkward.forms.form | None): If not none
```

Comment:
```suggestion
        form_mapping (Callable[awkward.forms.Form] -> awkward.forms.Form | None): If not none
```

Reply:
`awkward` is not loaded here (and it can't be, not at module level). However, I don't think this docstring depends on that.